### DISCUSSION TITLE
Dynamo + Grasshopper Installations failed

### DISCUSSION THREAD

**Discussion by bger-dhoward** [BHoM/BHoM#1487] [Q&A]
I ran the BHoM_v6.0.beta.0.msi installer. I see the BHoM tab in my Revit and Excel ribbons now, but I don't see a loaded package in Dynamo or in Grasshopper. Are any additional actions required to get these loaded, or did the installation just fail? Are there alternative ways of loading these components, since it appears the installation was partially successful (Revit + Excel).

Not sure if I need to be an admin on my machine to get the installation to work, but I didn't receive any notifications indicating that this was required.

I will attempt to build from source, but it would be good to know if this is going to need to be done to deploy to anybody else in my office.

Also, it would be helpful for the documentation to include a section on what you should see with a successful installation using the msi, how to check that different components installed correctly, and/or a list of all programs that the installer is going to attempt to add-on to.

**Comment by travispotterBH** [comment]
Hi @bger-dhoward. Quick question. Did you have Rhino open when you installed? Would you be open to trying to install from the beta msi file with Revit/Excel/Rhino processes fully closed? https://bhom.xyz/

**Reply by bger-dhoward** [reply]
I ran the latest v6.1 msi installer, and now I'm seeing the BHoM tools in Grasshopper, but not in Dynamo (both 2.12 for Revit 2022, and 2.16 for Revit 2023).  I had no programs open when I ran the msi installer.

I believed I had Revit and Rhino both closed when I ran the v6.0 installer when I posted the original comment above.

**Comment by RicardoWatson** [comment]
did you ever get this fixed as I am experiencingthe same issue with Dynamo 2.16 for Revit 2023. It looks as though it's installing folders for 2.0 and 2.3. I did notice that the folders in \AppData\Roaming\Dynamo\Dynamo Revit\2.3\Packages\BHOM\ did not have a dyf folder - should they?

**Comment by FraserGreenroyd** [comment]
Hi @RicardoWatson and @bger-dhoward , sorry I missed this conversation earlier in the year and didn't realise it was unresolved.

I ended up responding on [this issue](https://github.com/BHoM/Dynamo_UI/issues/324#issuecomment-1689767867) which basically highlighted that the last version of Dynamo which BHoM tried to be compatible with was 2.14, with it being fairly certain not to work with Revit 2023.

As such, Dynamo Toolkit isn't one of the UIs we're currently actively maintaining due to various legacy issues with Dynamo plugins and resource constraints, but is something we could take another look at if more recent versions of Dynamo are more stable for us to reflect the code base into (in a manner similar to Grasshopper).

From the 7.0 beta, coming out later this month (hopefully), installation for Dynamo has been removed, due to the current maintenance headache we have with it - but we'd be up for revisiting that decision in 2024 if the code base has become a bit more stable for us to work with.

However, in answer to this thread, I don't think BHoM will work for Dynamo 2.16 and may not work for Dynamo at all currently 😞
